### PR TITLE
minor additions to servant and soft refresh

### PR DESCRIPTION
**Pull request by 0x1f415**
servant: add tag checking functionality
soft refresh: option to trigger refresh when the period key is pressed

### CODE REVIEW

**Comment by 0x1f415** [comment]
ugh, I'll fix the mixed indentation problem in a minute.

**Comment by hobinjk** [comment]
You should also make sure to include version number increases in the first few lines of servant.js and soft_refresh.js.

File: Extensions/soft_refresh.js [hunk 1]
```diff
@@ -24,6 +24,11 @@ XKit.extensions.soft_refresh = new Object({
 			default:true,
 			value:true
 		},
+		"use_period_key":{
+			text:"Soft refresh when the period key is pressed.",
+			default:false,
+			value:true
```
Comment (hobinjk) [inline Extensions/soft_refresh.js]: value should be `false` to agree with `default: false`

File: Extensions/soft_refresh.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 //* TITLE Soft Refresh **//
-//* VERSION 0.5.5 **//
+//* VERSION 0.6 **//
```
Comment (hobinjk) [inline Extensions/soft_refresh.js]: This should be 0.6.0 instead of 0.6.

**Comment by hobinjk** [comment]
@coderpatsy The Servant functionality works for me on the search page's posts.

**Comment by 0x1f415** [comment]
I don't have any motivation to fix this anymore (I no longer have a use for the feature) so I'm closing this.

**Pull request closed without merging**